pgsync/sync.py

```diff
@@ -1116,7 +1128,8 @@ def sync(
 
         count: int = self.fetchcount(node._subquery)
 
-        logger.debug(f"sync started: {count} events to sync")
+        logger.debug(f"sync started: {count} documents to sync")
+        batch_size: int = 100
```

Comment:
```python
        log_every: int = 100
```
we are not batching it, and we we name it, log it as batch size it gives a false impression that documents are synced in 100s, which they are not